arch/xtensa/src/esp32s3/esp32s3_spi.c

```diff
@@ -1312,22 +1337,23 @@ void esp32s3_spi_dma_init(struct spi_dev_s *dev)
 {
   struct esp32s3_spi_priv_s *priv = (struct esp32s3_spi_priv_s *)dev;
 
-  /* Enable GDMA clock for the SPI peripheral */
-
-  modifyreg32(SYSTEM_PERIP_CLK_EN0_REG, 0, priv->config->dma_clk_bit);
-
-  /* Reset GDMA for the SPI peripheral */
-
-  modifyreg32(SYSTEM_PERIP_RST_EN0_REG, priv->config->dma_rst_bit, 0);
-
```

Comment:
Why did you remove device initialization? I recommend reverting it...

Reply:
In technical reference, There aren't these two registers. So i delete it.

Reply:
It's the overall peripheral initialization. You can keep it...

Reply:
Please, revert your changes and keep the overall device initialization.

Reply:
[https://www.espressif.com.cn/sites/default/files/documentation/esp32-s3_technical_reference_manual_en.pdf#page=810](url)
But, in SYSTEM_PERIP_CLK_EN0_REG, bit 22 and bit 27 are reserved. In SYSTEM_PERIP_CLK_EN1_REG, bit 22 and bit 27 are reserved either. In function ` esp32s3_dma_init() ` , dma clock will be enabled and reset status will be removed.

Reply:
> Please, revert your changes and keep the overall device initialization.

OK